package.json

```diff
@@ -39,29 +34,30 @@
   },
   "homepage": "https://github.com/binary-com/deriv-api#readme",
   "devDependencies": {
-    "@babel/core": "^7.4.5",
-    "@babel/plugin-transform-runtime": "^7.6.0",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/runtime": "^7.6.0",
-    "babel-jest": "^24.8.0",
-    "babel-loader": "^8.0.6",
-    "codecov": "^3.6.5",
-    "documentation": "^12.2.0",
-    "eslint": "^6.1.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-align-assignments": "^1.0.0",
-    "eslint-plugin-arca": "^0.8.1",
-    "eslint-plugin-import": "^2.18.0",
-    "husky": "^2.4.1",
-    "jest": "^25.2.7",
-    "webpack": "^4.42.1",
-    "webpack-bundle-analyzer": "^3.6.1",
-    "webpack-cli": "^3.3.6",
-    "ws": "^7.1.2"
+    "@babel/core": "^7.15.0",
+    "@babel/plugin-transform-runtime": "^7.15.0",
+    "@babel/preset-env": "^7.15.0",
+    "@babel/runtime": "^7.14.8",
+    "babel-jest": "^27.0.6",
+    "babel-loader": "^8.2.2",
+    "codecov": "^3.8.3",
+    "documentation": "^13.2.5",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-align-assignments": "^1.1.2",
+    "eslint-plugin-arca": "^0.10.1",
+    "eslint-plugin-import": "^2.24.0",
+    "husky": "^7.0.1",
+    "jest": "^27.0.6",
+    "webpack": "^5.49.0",
+    "webpack-bundle-analyzer": "^4.4.2",
+    "webpack-cli": "^4.7.2",
+    "webpack-node-externals": "^3.0.0"
   },
   "dependencies": {
-    "dayjs": "^1.8.15",
+    "dayjs": "^1.10.6",
     "json-stable-stringify": "^1.0.1",
-    "rxjs": "^6.5.2"
+    "rxjs": "^7.3.0",
+    "ws": "^8.0.0"
```

Comment:
why do you need websockets? Seems it was not needed before

Reply:
Yes you are right. I introduced it by mistake.

Reply:
The problem is here : https://github.com/binary-com/deriv-api/blob/master/src/deriv_api/DerivAPIBasic.js#L110

test will fail without this line. But if we have this line, then I think FE side will break. what should we do ?

Reply:
https://www.npmjs.com/package/ws    
```
Note: This module does not work in the browser. The client in the docs is a reference to a back end with the role of a client in the WebSocket communication. Browser clients must use the native WebSocket object. To make the same code work seamlessly on Node.js and the browser, you can use one of the many wrappers available on npm, like isomorphic-ws.

```

Reply:
OK I got it. there is a line https://github.com/binary-com/deriv-api/blob/master/examples/a_simple_bot.js#L1  
```
global.WebSocket = require('ws');
```
I think it is not a good idea because user will be surprised and be forced to mantain the dependence manually. (and I even suspect it will not play well with the upgraded dep modules). Let me test and maybe try https://github.com/heineiuo/isomorphic-ws